### PR TITLE
feat(Lambda): Add invokeLambdaFunction utility function

### PR DESCRIPTION
**Pull request by bayoudhi**

### CODE REVIEW

File: src/utils/lambda.ts [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   */
+  payload: object;
+  /**
+   * An optional Lambda SDK client configuration.
+   */
+  clientConfig?: LambdaClientConfig;
+};
+
+/**
+ * Invoke a lambda function.
```
Comment (bboure) [inline src/utils/lambda.ts]: `Lambda`

**Review by bboure — APPROVED** [review]
LGTM.
simple enough

File: src/utils/lambda.ts (hunk 1 above)
Comment (bayoudhi) [reply src/utils/lambda.ts]:
```suggestion
 * Invoke a Lambda function.
```

**Comment by bboure** [comment]
We still miss the doc, right?

**Comment by bayoudhi** [comment]
> We still miss the doc, right?

Yes, I'll work on it.

**Pull request merged**